src/object/schedcontext.c

```diff
@@ -284,6 +284,12 @@ void schedContext_unbindTCB(sched_context_t *sc, tcb_t *tcb)
     tcbSchedDequeue(sc->scTcb);
     tcbReleaseRemove(sc->scTcb);
 
+    /* If the unbound TCB has a timeout handler, it is triggered to
```

Comment:
nit: the commit message on this one should be 'Timeout fault on SC unbind'. SC unbind itself doesn't have a timeout.

Reply:
also forcibly is a typo in the commit message

Reply:
Fixed this